projects/1.19/assets/ars-nouveau-extended-glyphs/ars_extended_glyphs/lang/zh_cn.json

```
@@ -0,0 +1,92 @@
+	"ars_extended_glyphs.glyph_name.glyph_createsource": "召唤魔源",
+	"ars_extended_glyphs.glyph_desc.glyph_createsource": "生成魔源，并注入6格之内的魔源容器中。强化与抑制会改变生成魔源的量。",
+	"ars_extended_glyphs.glyph_name.glyph_herobrine": "召唤Herobrine",
+	"ars_extended_glyphs.glyph_desc.glyph_herobrine": "",
+	"ars_extended_glyphs.glyph_name.glyph_repair": "修复",
+	"ars_extended_glyphs.glyph_desc.glyph_repair": "若受影响目标主手中持有工具，则将其耐久度重置。",
+	"ars_extended_glyphs.glyph_name.glyph_targetself": "瞄准自身",
+	"ars_extended_glyphs.glyph_desc.glyph_targetself": "使得后续的法术效果作用于施法者。",
+	"ars_extended_glyphs.glyph_name.glyph_returnhome": "返回重生点",
+	"ars_extended_glyphs.glyph_desc.glyph_returnhome": "将受影响的玩家传送回他们的重生点。对其他生物和未设置过重生点的玩家无效。",
+	"ars_extended_glyphs.glyph_name.glyph_sethome": "设置重生点",
+	"ars_extended_glyphs.glyph_desc.glyph_sethome": "将施法者的重生点设置在其所处的位置。",
+	"ars_extended_glyphs.glyph_name.glyph_conjurelava": "召唤熔岩",
+	"ars_extended_glyphs.glyph_desc.glyph_conjurelava": "在目标位置放置熔岩。",
+	"ars_extended_glyphs.glyph_name.glyph_atrophy": "萎缩",
+	"ars_extended_glyphs.glyph_desc.glyph_atrophy": "致使目标受到一定时间的虚弱。",
+	"ars_extended_glyphs.glyph_name.glyph_fertility": "丰产",
+	"ars_extended_glyphs.glyph_desc.glyph_fertility": "仅对动物有效。如果是幼年动物，则加快其生长；否则重置交配冷却。",
+
+	"item.ars_extended_glyphs.magestone": "魔石",
+	"item.ars_extended_glyphs.refined_magestone": "精炼魔石",
+
+	"item.ars_extended_glyphs.ring_of_greatest_discount": "高等节魔之戒",
+
+	"item.ars_extended_glyphs.music_disc_record": "音乐唱片",
+	"item.ars_extended_glyphs.music_disc_record.desc": "1nsider - rerecord",
+
+	"item.ars_extended_glyphs.blank_tablet": "远古石板",
+	"item.ars_extended_glyphs.air_tablet": "气之石板",
+	"item.ars_extended_glyphs.water_tablet": "水之石板",
+	"item.ars_extended_glyphs.earth_tablet": "土之石板",
+	"item.ars_extended_glyphs.fire_tablet": "火之石板",
+	"item.ars_extended_glyphs.abjuration_tablet": "防护之石板",
+	"item.ars_extended_glyphs.conjuration_tablet": "召唤之石板",
+	"item.ars_extended_glyphs.manipulation_tablet": "操纵之石板",
+
+	"item.ars_extended_glyphs.blank_tablet_fragment": "无趣饰件",
+	"item.ars_extended_glyphs.air_tablet_fragment": "气之饰件",
+	"item.ars_extended_glyphs.fire_tablet_fragment": "火之饰件",
+	"item.ars_extended_glyphs.water_tablet_fragment": "水之饰件",
+	"item.ars_extended_glyphs.earth_tablet_fragment": "土之饰件",
+
+	"item.ars_extended_glyphs.ritual_difficulty_increase": "难度",
+	"ars_extended_glyphs.ritual_desc.ritual_difficulty_increase": "将世界难度提升一级。使用羽毛增强可换为降低。",
+
+	"itemGroup.ars_extended_glyphs": "Extended Glyphs",
+
+	"tooltip.aeg.await": "按住左Shift查看更多信息。",
+
+	"tooltip.aeg.blank_tablet": "自久远过去而来的神秘石板……",
+	"tooltip.aeg.blank_tablet_f": "可在附魔装置处制作为抗性护符。",
+	"tooltip.aeg.tablet": "装备时将该学派魔符的魔力消耗降低50点。",
+	"tooltip.aeg.tablet_warning": "获得时自带绑定诅咒！",
+	"tooltip.refined_magestone": "可用作法术核心。",
+	"tooltip2.refined_magestone": "堆叠数为16，总计可使最大魔力+80。",
+
+	"tooltip.aeg.tablet_f.air": "装备时会消耗魔力抵消摔落伤害。",
+	"tooltip.aeg.tablet_f.fire": "装备时会消耗魔力抵消火焰和熔岩造成的伤害。",
+	"tooltip.aeg.tablet_f.earth": "装备时会消耗魔力抵消窒息、鞘翅动能，以及地表灼热造成的伤害。",

```

```suggestion
	"tooltip.aeg.tablet_f.earth": "装备时会消耗魔力抵消窒息、鞘翅动能，以及灼热方块造成的伤害。",

```
指的大概是岩浆块，我不确定营火能否伤害。

---

营火不防